packages/components/CollectionView.tsx

```diff
@@ -15,24 +14,35 @@ import { GradientText } from "./gradientText";
 type CollectionViewSize = "XL" | "XS";
 export const COLLECTION_VIEW_SM_WIDTH = 124;
 export const COLLECTION_VIEW_SM_HEIGHT = 164;
-export const COLLECTION_VIEW_XL_WIDTH = 196;
-export const COLLECTION_VIEW_XL_HEIGHT = 266;
+export const COLLECTION_VIEW_XL_WIDTH = 256;
+export const COLLECTION_VIEW_XL_HEIGHT = 315;
 
 export const CollectionView: React.FC<{
   item: Collection;
   size?: CollectionViewSize;
   linkToMint?: boolean;
-}> = ({ item, size = "XL", linkToMint }) => {
-  const [, creatorAddress] = parseUserId(item.creatorId);
-  const userInfo = useNSUserInfo(item.creatorId);
+  mintState: number;
+}> = ({ item, size = "XL", linkToMint, mintState }) => {
   const navigateToCollection = useNavigateToCollection(item.id, {
     forceSecondaryDuringMint: item.secondaryDuringMint,
     forceLinkToMint: linkToMint,
   });
   const sizedStyles = useMemo(() => StyleSheet.flatten(styles[size]), [size]);
 
+  const navigateToTwitter = () => {
+    Linking.openURL(item.twitterUrl);
+  };
+  const { info } = useCollectionInfo(item.id);
```

Comment:
@n0izn0iz what would be the best way to batch these requests?
Do I need to make a new endpoint in the backend?